.github/workflows/ci.yml

```diff
@@ -266,7 +266,7 @@ jobs:
 
   trigger-flox-installers-workflow:
     name: "Build installers"
-    if: ${{ github.base_ref == 'refs/heads/main' && github.event_name == 'push' }}
+    if: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' }}
```

Comment:
It looks like we changed this to build tags in 57f544b92146d30133ad6255e76a3d08fc7d066a.

Do you know if that still works after this change?